packages/network-controller/src/NetworkController.ts

```diff
@@ -187,7 +241,7 @@ export class NetworkController extends BaseControllerV2<
 
   private initializeProvider(
     type: NetworkType,
-    rpcTarget?: string,
+    rpcUrl?: string,
```

Comment:
It may or may not be worth it to make this change now, but we use `rpcUrl` in the extension and we will obviously need to land on one before we merge

Reply:
It makes sense to use `rpcUrl` in the network config, but we shouldn't change this in the provider config object yet if we can. Eventually I think we ought to merge the provider config and network config, so perhaps when we do that, this problem will just go away.

Reply:
yeah I was wondering if changing it now would help with the merge since the provider extension calls it `rpcUrl`? But I could see it either way!

Reply:
@Gudahtt what do you think on this?

Reply:
Ah, I hadn't realized. That makes sense to change before we merge the controllers then. However maybe we ought to do this in a new PR if it's not too much trouble? It seems like the sort of thing that could get missed.

Reply:
> It seems like the sort of thing that could get missed.

What do you mean by this? Do you mean I might not have caught all instances? Or that reviewers won't pay sufficient attention to the change?

Reply:
Ok yeah that makes sense! I'll rollback this name change

Reply:
rolled back this name change here: https://github.com/MetaMask/core/pull/1064/commits/611cbde7c55ea76cf8ad7c0610b91fbb053f0b51